File: pset6/mario.py
import cs50

height = 0
while True:
    height = cs50.get_int()
    if height >= 0 and height <= 23:
        break
width = height + 1
for i in range(height):
    print(" "*(height - i - 1), end="")
    print("#"*(i + 2))

File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        printf("Height: ");
        n = get_int();
    }
    while (n < 0 || n > 23);
    for (int i = 0; i < n; i++)
    {
        int j;
        for (j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (; j < n + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}